webapp/Connector/src/view/Learn.js

```diff
@@ -541,7 +541,7 @@ Ext.define('Connector.view.Learn', {
                 }
             }
             else {
-                if(store.getCount() === 0)
+                if(store.getCount() === 0 && !store.isFiltered())
```

Comment:
Needed to correctly handle events where the store was being filtered, versus being uninitialized.